bumble/core.py

```diff
@@ -288,8 +288,19 @@ def __str__(self):
 
         return result
 
-    def __repr__(self):
-        return str(self)
+    def __repr__(self) -> str:
```

Comment:
Could you add a couple of test cases for this

Reply:
I'm not sure what to test here, it's copy/pasted from `to_hex_str` with slightly modifications

Reply:
I'd just do something like:
```
  def test_uuid_repr():
    assert UUID("1234").__repr__() == "1234"
    assert UUID("df5ce654-e059-11ed-b5ea-0242ac120002").__repr__() == "DF5CE654-E059-11ED-B5EA-0242AC120002"

Reply:
Ack :+1: